lucene/core/src/java/org/apache/lucene/codecs/lucene90/Lucene90DocValuesConsumer.java

```diff
@@ -496,6 +516,25 @@ public void addSortedField(FieldInfo field, DocValuesProducer valuesProducer) th
 
   private void doAddSortedField(FieldInfo field, DocValuesProducer valuesProducer)
       throws IOException {
+
+    int numDocsWithValue = 0;
+    MinMaxTracker blockMinMax = new MinMaxTracker();
+
+    SortedDocValues sorted = valuesProducer.getSorted(field);
+    for (int doc = sorted.nextDoc(); doc < DocIdSetIterator.NO_MORE_DOCS; doc = sorted.nextDoc()) {
+      final int v = sorted.ordValue();
+      blockMinMax.update(v);
+      if (blockMinMax.numValues == NUMERIC_BLOCK_SIZE) {
+        blockMinMax.nextBlock();
+      }
+      numDocsWithValue++;
+    }
+    blockMinMax.finish();
+
```

Comment:
Maybe a comment for a future reader lacking context?

```suggestion

// ordinals will always have a min value of 0 and a max of length - 1:
```